### PR TITLE
Refactor po file parsing and update cache files

### PR DESCRIPTION
**Pull request by pmor**
This addresses issue #57 by refactoring the `.po` file parsing in `LiveDataProvider` to ignore "fuzzy" strings.

Additionally, it also changes `LiveDataProvider` to find the translations from the alpha3 codes instead of the english `msgid` lines. This also allows to ensure we use countries' "common names" when they are available.

Cache files have already been updated to remove the incorrect translations caused by the "fuzzy" strings in the `.poi` files

### CODE REVIEW

**Comment by grosser** [comment]
looks like we are losing some good translations too (but hard for me to judge how good they really were, they just look sane ...), but overall makes sense.
Will this change the public api too in any way ?

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,4 @@ gemspec
 gem 'bump'
 gem 'rake'
 gem 'rspec', '~>3.0'
+gem 'simple_po_parser'
```
[the remark this answers is not in the document]
Comment (pmor) [reply Gemfile]: db4196efa880a1a7fc8230ec3b2121d230689ec1

**Comment by pmor** [comment]
It shouldn't change anything in the public API, the changes are essentially limited to LiveDataProvider's private methods

**Comment by grosser** [comment]
0.15.0 🤞

**Pull request merged**